CMakeLists.txt

```diff
@@ -16,7 +16,7 @@
 # along with this program; if not, see <http://www.gnu.org/licenses/>.
 #============================================================================
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.1...3.31)
```

Comment:
I don't understand how a minimum can be a range, I'd like to have a comment here explaining the semantics